docs/user-guide/install.md

```diff
@@ -1,16 +1,13 @@
-# Introduction
-
-vSphere Docker Volume Service is simple to install. It has zero configuration and and zero credential management post install.
-
-[Tagged releases](https://github.com/vmware/docker-volume-vsphere/releases) include the software bundles to install on ESX and on the VM.
-
-In addition the ```make build-all``` will generate the packages.
+###...
+# Installation
 
+The installation has two parts – installation of the backend on ESX and installation of Docker plugin on the hosts where you plan to run containers with storage needs. You can use the [Tagged releases](https://github.com/vmware/docker-volume-vsphere/releases) in Github to get latest version or pick up a specific version. 
+ 
 # Install on ESX [![VIB_Download](https://api.bintray.com/packages/vmware/vDVS/VIB/images/download.svg)](https://bintray.com/vmware/vDVS/VIB/_latestVersion)
 
-[VIB](http://blogs.vmware.com/vsphere/2011/09/whats-in-a-vib.html) and [Offline Depot](https://pubs.vmware.com/vsphere-60/index.jsp#com.vmware.vsphere.install.doc/GUID-29491174-238E-4708-A78F-8FE95156D6A3.html?resultof=%2522%256f%2566%2566%256c%2569%256e%2565%2522%2520%2522%256f%2566%2566%256c%2569%256e%2522%2520%2522%2564%2565%2570%256f%2574%2522%2520) are the packages built to install the backend for the service on ESX. The backend can be installed using esxcli or vmware tools such as [VUM](http://pubs.vmware.com/vsphere-60/topic/com.vmware.ICbase/PDF/vsphere-update-manager-601-install-administration-guide.pdf)
+The backend for services can be installed on ESX using exscli or vmware tools such as VUM (Hyperlink to VUM). You would need the [VIB](http://blogs.vmware.com/vsphere/2011/09/whats-in-a-vib.html) and [Offline Depot](https://pubs.vmware.com/vsphere-60/index.jsp#com.vmware.vsphere.install.doc/GUID-29491174-238E-4708-A78F-8FE95156D6A3.html?resultof=%2522%256f%2566%2566%256c%2569%256e%2565%2522%2520%2522%256f%2566%2566%256c%2569%256e%2522%2520%2522%2564%2565%2570%256f%2574%2522%2520) packages to install the backend. 
 
-Log into ESXi host and download the [latest release](https://bintray.com/vmware/vDVS/VIB/_latestVersion) of vDVS driver VIB on ESXi and initiate the install by specifying the full path to the VIB.
+We will demonstrate how to install the packages using Excli here, you can download the [latest release](https://bintray.com/vmware/vDVS/VIB/_latestVersion) of vDVS driver VIB on the ESXi. Assuming that you have downloaded the VIB at /tmp location you can run the below command to install it on ESXi
```

Comment:
Original version (Log into..) of this statement is better so we can revert to it.